Source/utils/display.cpp

```diff
@@ -243,11 +243,13 @@ bool SpawnWindow(const char *lpWindowName)
 		flags |= SDL_WINDOW_FULLSCREEN;
 	}
 
-	if (*sgOptions.Gameplay.grabInput) {
-		flags |= SDL_WINDOW_INPUT_GRABBED;
-	}
-
 	ghMainWnd = SDL_CreateWindow(lpWindowName, SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, windowSize.width, windowSize.height, flags);
+
+    // Note: https://github.com/libsdl-org/SDL/issues/962

```

Comment:
Looks like these 2 comment lines are indented with spaces instead of a tab, causing `clang-format` check to fail

Reply:
Corrected comment.